src/main.rs

```diff
@@ -47,32 +11,14 @@ fn calculate_score(ortho: &Ortho) -> (usize, usize) {
 
 fn main() -> Result<(), FoldError> {
     let input_dir = "./fold_state/input";
+    let output_dir = "./fold_state/output";
```

Comment:
Do not write these to a separate output space. Take in txt, put out .bin, remove txt, and scan again for the next txt. Don't batch the txt scan - search for txt in a loop and stop when all are gone.